packages/flutter/lib/src/material/stepper.dart

```diff
@@ -140,6 +140,7 @@ class Stepper extends StatefulWidget {
     this.onStepTapped,
     this.onStepContinue,
     this.onStepCancel,
+    this.controlsBuilder,
```

Comment:
I would not use [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) for a constructor parameter, instead maybe just call it `controls`?

Reply:
I take this back, we use `fooBuilder` everywhere ☹️